benchexec/tablegenerator/react-table/src/components/LinkOverlay.js

```diff
@@ -148,6 +155,13 @@ export default class LinkOverlay extends React.Component {
     }
   };
 
+  getScrollBarWidth() {
+    const modalBox =
+      window.document.body.lastElementChild.lastElementChild.lastElementChild;
```

Comment:
This looks error-prone and not very robust. Is there no better way to get the desired DOM element, e.g., via an id?

Reply:
Now it gets desired DOM element via class name, because there is no id in the desired DOM element.